support-frontend/assets/components/checkoutBenefits/checkoutBenefitsList.tsx

```diff
@@ -72,34 +76,58 @@ const para = css`
 export type CheckoutBenefitsListProps = {
 	title: string;
 	checkListData: CheckListData[];
+	contributionType: ContributionType;
+	countryGroupId: CountryGroupId;
 	buttonCopy: string | null;
+	nudgeTitleTopCopy: string;
+	nudgeTitleCopy: string;
```

Comment:
How about something like?

```suggestion
	nudgeTitleCopySection1: string;
	nudgeTitleCopySection2: string;
```
Something that indicates that these are 2 parts of a whole would be great here

Reply:
Yes better group naming